ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/graphql/internal/pojoadapter/BlockAdapterBase.java

```diff
@@ -142,16 +142,12 @@ public Long getNumber() {
   }
 
   public AccountAdapter getAccount(final DataFetchingEnvironment environment) {
-
     final BlockchainQueries query = getBlockchainQueries(environment);
     final long bn = header.getNumber();
+    final Address address = environment.getArgument("address");
```

Comment:
Is there a reason to take this variable outside of the closure scope?

Reply:
No strong reason it was to make the Function of getAndMapWorldState as clean as possible. Happy to move this code back to it's original place if you think it's not worth keep the function more readable.